clients/web/src/stylesheets/layout/globalNav.styl

```diff
@@ -3,16 +3,13 @@ $emptyBorder=1px solid transparent
 $minHeight=500px
 
 .g-global-nav-main
-  box-shadow inset -1px 0 0 #eee
+  -webkit-box-shadow inset 0 -35px 45px -10px rgba(255, 255, 255, 1)
```

Comment:
The Nib CSS utility library (which Girder always [imports via Stylus](https://github.com/girder/girder/blob/ba1544a009c6abf437a82810fa1a9f0529c16fa9/grunt_tasks/webpack.config.js#L105) should [provide vendor prefixes for `box-shadow` automatically](https://github.com/stylus/nib/tree/master/docs#transparent-mixins), so the rule should only need to defined once.

Reply:
Thanks Brian, This is good to know moving forward!